apps/admin-gui/src/app/app.component.ts

```diff
@@ -46,6 +51,29 @@ export class AppComponent implements OnInit {
 
   ngOnInit(): void {
     this.principal = this.store.getPerunPrincipal();
+    interval(30000).subscribe(() => {
```

Comment:
so I we would assume that your solution is working, I dont think that I would like to check every 30 sec for new version and  if yes, just reload app. What if the user have some form half filled and then suddenly the gui reloads. That would be super anoying. I think we would like it just on the start of the application. 
Another viable solution, I think, could be checking the version when user enters component "user-dashboard" in the onInit method in that component
what do you think @Vojtech-Sassmann

Reply:
If we checked for a new version just on the start of the app, users who already had the app opened, would not be using the latest version, in theory, for the rest of the day.

Reply:
Checking for the new version every 30s might be a viable solution; the only problem I can see is the reload. We should inform users that we have released a new version and that they can reload the application. This should also be done across the whole application, not only the home screen since some users might be idling on a different page.

Reply:
yep, agree with vojta, there should be dialog if the loadAppVersion detected changes in version. if there is detected change in version dialog should appear - in the title should be something like: "New version was detected. You must reload browser to load new version. ... " and two buttons: 1. "Reload GUI" 2. "Got it! I will reload GUI later"
something like windows has when they want to update
Also on the start of the application if the change in the version is detected then no dialog should appear and the gui should automatically reload